tests/post_training/model_scope.py

```diff
@@ -218,7 +218,18 @@
             "preset": QuantizationPreset.MIXED,
             "model_type": ModelType.TRANSFORMER,
         },
-        "backends": ALL_PTQ_BACKENDS,
+        "backends": [BackendType.OV],
+    },
+    {
+        "reported_name": "timm/swin_base_patch4_window7_224_no_sq",
+        "model_id": "swin_base_patch4_window7_224",
+        "pipeline_cls": ImageClassificationTimm,
+        "ptq_params": {
+            "preset": QuantizationPreset.MIXED,
+            "model_type": ModelType.TRANSFORMER,
+            "advanced_parameters": AdvancedQuantizationParameters(smooth_quant_alpha=-1.0),
```

Comment:
Please, use the `smooth_quant_alphas=AdvancedSmoothQuantParameters(matmul=-1)`

Reply:
Build post_training_quantization/260 to check new configurations is successful